troubadix/standalone_plugins/no_solution.py

```diff
@@ -0,0 +1,276 @@
+    return parser.parse_args()
+
+
+def check_skip_script(file_content: str) -> bool:
+    solution_type = SOLUTION_TYPE_PATTERN.search(file_content)
+    if (
+        solution_type
+        and solution_type.group("value") != SOLUTION_TYPE_NONE_AVAILABLE
+    ):
+        return True
+
+    cvss = CVSS_PATTERN.search(file_content)
+    if cvss and cvss.group("value") == CVSS_DETECTION_SCRIPT:
+        return True
+
+    return False
+
+
+def extract_tags(content: str) -> Optional[Tuple[datetime, datetime, str]]:
+    solution_match = SOLUTION_PATTERN.search(content)
+    if not solution_match:
+        return None
+
+    date_match = SOLUTION_DATE_PATTERN.search(solution_match.group("value"))
+    if not date_match:
+        return None
+
+    solution_date = parse_solution_date(date_match.group("date"))
+    if not solution_date:
+        return None
+
+    creation_match = CREATION_DATE_PATTERN.search(content)
+    if not creation_match:
+        return None
+
+    creation_date = datetime.strptime(
+        creation_match.group("value")[:10], CREATION_DATE_FORMAT
+    )
+
+    oid_match = OID_PATTERN.search(content)
+    if not oid_match:
+        return None
+
+    oid = oid_match.group("value")
+
+    return solution_date, creation_date, oid
+
+
+def check_no_solutions(
+    files: Iterable[Path], milestones: List[int], snooze_duration: int
+) -> List[Tuple[int, List[Tuple[Path, str, datetime, datetime]]]]:
+    summary = defaultdict(list)
+
+    for nasl_file in files:
+        content = nasl_file.read_text(encoding=CURRENT_ENCODING)
+
+        if check_skip_script(content):
+            continue
+
+        extracted_tags = extract_tags(content)
+        if not extracted_tags:
+            raise ValueError(f"Could not extract tags from {nasl_file}")
+
+        solution_date, creation_date, oid = extracted_tags
+
+        solution_diff = datetime.now() - solution_date
+
+        if solution_diff < timedelta(days=snooze_duration * MONTH_AS_DAYS):
+            continue
+
+        creation_diff = datetime.now() - creation_date
+
+        for milestone in milestones:
+            milestone_delta = timedelta(days=milestone * MONTH_AS_DAYS)
+            if creation_diff >= milestone_delta:
+                summary[milestone].append(
+                    (nasl_file, oid, solution_date, creation_date)
+                )
+                break
+
+    return sorted(
+        [(milestone, vts) for milestone, vts, in summary.items()],
+        key=lambda tuple: tuple[0],
+        reverse=True,
+    )
+
+
+def print_info(
+    term: ConsoleTerminal, milestones: List[int], threshold: int, root: Path
+):
+    term.bold_info("Reported VTs with solution type 'NoneAvailable'")
+    with term.indent():
+        term.print(f"Root directory: {root}")
+        milestone_str = ", ".join(str(milestone) for milestone in milestones)
+        term.print(f"Milestones: {milestone_str} months")
+        term.print(f"Expect no solution threshold: {threshold} months")
+
```

Comment:
```suggestion
        term.print(f"Snooze duration: {snooze_duration} months")
```
Can you print the snooze duration, just like the other arguments?

Reply:
Fixex